lib/index.js

```diff
@@ -170,7 +170,7 @@ class UI {
     }
 
     if (typeof error === 'object' && error !== null && !error.isSilentError) {
-      if (process.env.CI) {
```

Comment:
This seems reasonable, but to ensure compatibility i believe `this.ci` should default to `process.env.CI` if `options.ci` is unset. With that change + associated test, I feel comfortable moving forward.

Reply:
Sounds reasonable will update the PR.